### DISCUSSION TITLE
posts/2022/02/thinking-about-syncing/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#9] [Announcements]
# posts/2022/02/thinking-about-syncing/

Some opinions I’ve formed while doing a semi-deep dive into cloud sync on macOS.

https://www.brycewray.com/posts/2022/02/thinking-about-syncing/

**Comment by jasikpark** [comment]
I certainly wish that there was a solid way to export all of your iCloud images to disk except by having enough space to sync your entire library to disk...

**Comment by codepoet80** [comment]
I've been using OneDrive on macOS for more than half a decade, and using it with two accounts (business and personal) for the past 3. Its always worked flawlessly, and its priced better than most alternatives.
Recently switched to OwnCloud for personal syncing, which had a bit of a learning curve, but is now working quite well. Still use OneDrive for work, still have no complaints.

**Reply by brycewray** [reply]
I’m glad OneDrive worked for you! However, I never was that fortunate; I kept getting odd glitches — including some Microsoft had no luck in diagnosing — and finally decided it wasn’t worth the trouble, especially once I no longer was using a workplace Windows environment to view both my files and work files, as had once been the case.

As for OwnCloud and similar methods: I have read about them out of curiosity but ended up reaching the conclusion my use cases aren’t really the ones for which they’re intended.
